apps/foundation-demo/src/app/checkout/components/shipping-option/shipping-option.component.html

```diff
@@ -0,0 +1,15 @@
+<fieldset class="shipping-option">
```

Comment:
This should should take a `ShippingOption[]` as data and loop over it to render the data.

Reply:
done